src/app/core/entity/entity-config.service.ts

```diff
@@ -123,4 +124,9 @@ export interface EntityConfig {
    * icon used to visualize the entity type
    */
   icon?: string;
+
+  /**
+   * base route of views for this entity type
+   */
+  route?: string;
```

Comment:
Great idea. Could we use the chance to also use this for all entities? At the moment we have some places where we still create the route through `ENTITY_TYPE + ':' + id` (e.g. in the `EntityBlockComponent`) which is rather dangerous as it requires the route to be equal to the entity type.
With this approach we have much more flexibility and can make the routes however we want.

Reply:
good point actually. Done.